### DISCUSSION TITLE
Helperify FFUF / Massdns

### DISCUSSION THREAD

**Discussion by TheTechromancer** [blacklanternsecurity/bbot#858] [Ideas]
It would be nice to expose these brute-force functionalities as helpers. A module could request the helper by setting `deps_helpers`, and we would have a new directory of these helpers which each have an ansible playbook attached that will install the necessary binaries, i.e. `ffuf` or `massdns`.

**Comment by TheTechromancer** [comment]
httpx is another candidate for this, as it would significantly speed up our bucket enumeration and potentially other modules like ntlm.

EDIT: It would also allow any module to raise `HTTP_RESPONSE`, which would be pretty cool.

**Comment by liquidsec** [comment]
very outdated now
